src/renderer/components/ReactFlowBox.tsx

```diff
@@ -119,7 +119,14 @@ export const ReactFlowBox = memo(({ wrapperRef, nodeTypes, edgeTypes }: ReactFlo
 
         if (isSnapToGrid) {
             for (const n of displayNodes) {
-                if (!isSnappedToGrid(n.position, snapToGridAmount)) {
+                if (n.type === 'breakPoint') {
+                    const snapped = snapToGrid(n.position, snapToGridAmount);
+                    const pos = {
+                        x: snapped.x - (n.width ?? 0) / 2,
+                        y: snapped.y - (n.height ?? 0) / 2,
+                    };
+                    n.position = pos;
```

Comment:
You just changed what it means for a node to be snapped to grid. This will cause problems. E.g. the node formatting already makes assumptions about grid alignment, and this change will likely interfere with it.

Why not just move the visuals of the node with CSS?

Reply:
that sounds infinitely more complicated. If i do it that way, i then have to go and change the logic for calculating the edge lines and i would have to offset all those by however much we offset the breakpoints in CSS. I suppose i could try it... but that doesnt seem like an elegant solution to this

Reply:
![image](https://github.com/chaiNNer-org/chaiNNer/assets/34788790/e5e7ef1b-1bb8-443f-8f32-2bd8985221f3)

This is after auto-formatting. I honestly don't see an issue with this

Reply:
See #2577